Makefile

```diff
@@ -34,6 +41,10 @@ proto:
 		--go-grpc_out=Mgrpc/service_config/service_config.proto=/proto-gen/api:. \
 		idl/proto/worker.proto
 
+.PHONY: ui
+ui:
+	GOOS=linux GO111MODULE=on packr2 --ignore-imports
```

Comment:
what does this do?

Reply:
Packr is used to embed static files into our binary. The way it does this is by generating *.go files which have the static files as byte strings and can be imported through Packr and served.